src/main/java/core/basesyntax/dao/FruitDao.java

```diff
@@ -0,0 +1,76 @@
+package core.basesyntax.dao;
+
+import java.util.Objects;
+
+public class FruitDao {
+    private String name;
+    private int quantity;
+
+    public FruitDao(String name, int quantity) {
+        this.name = name;
+        this.quantity = quantity;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
```

Comment:
Consider adding validation in the `setName` method to ensure that the name is not null or empty. This will prevent invalid `FruitDao` objects from being created.